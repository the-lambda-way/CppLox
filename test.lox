var language = "lox";
print language;
print 1 + 2;
