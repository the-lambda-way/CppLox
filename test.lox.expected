VAR var empty
IDENTIFIER language empty
EQUAL = empty
STRING "lox" lox
SEMICOLON ; empty
PRINT print empty
IDENTIFIER language empty
SEMICOLON ; empty
PRINT print empty
NUMBER 1 1.000000
PLUS + empty
NUMBER 2 2.000000
SEMICOLON ; empty
END_OF_FILE  empty
